lib/java-wrapper.ts

```diff
@@ -36,13 +36,14 @@ async function runJavaCommand(
 }
 
 export async function getEntrypoints(targetPath: string): Promise<string[]> {
-  if (!(await exists(path.join(targetPath, 'target')))) {
-    throw new Error('Could not find target folder');
-  }
   const entrypointsFiles = await glob(
     path.join(targetPath, '**/target/classes/**/*.class'),
   );
 
+  if (entrypointsFiles.length == 0) {
+    throw new Error('Could not find a target folder');
```

Comment:
You have the same error here: https://github.com/snyk/java-call-graph-builder/blob/60a88e86776c577a6f0c3d11508a9b6b0059e2c7/lib/java-wrapper.ts#L88

Can we somehow limit this to just finding target folders?